app/views/RoomInfoView/Livechat.tsx

```diff
@@ -50,37 +51,38 @@ const Livechat = ({ room, roomUser, theme }) => {
 	return (
 		<>
 			<Title title={I18n.t('User')} theme={theme} />
-			<Timezone utcOffset={roomUser.utc} theme={theme} />
-			<Item label={I18n.t('Username')} content={roomUser.username} theme={theme} />
+			{/* TODO: Check if this param is outdated */}
+			{/* @ts-ignore */}
+			<Timezone utcOffset={roomUser.utc} />
+			<Item label={I18n.t('Username')} content={roomUser.username} />
 			<Item
 				label={I18n.t('Email')}
 				content={roomUser.visitorEmails?.map(email => email.address).reduce((ret, item) => `${ret}${item}\n`)}
-				theme={theme}
 			/>
 			<Item
 				label={I18n.t('Phone')}
 				content={roomUser.phone?.map(phone => phone.phoneNumber).reduce((ret, item) => `${ret}${item}\n`)}
-				theme={theme}
 			/>
-			<Item label={I18n.t('IP')} content={roomUser.ip} theme={theme} />
-			<Item label={I18n.t('OS')} content={roomUser.os} theme={theme} />
-			<Item label={I18n.t('Browser')} content={roomUser.browser} theme={theme} />
-			<CustomFields customFields={roomUser.livechatData} theme={theme} />
+			<Item label={I18n.t('IP')} content={roomUser.ip} />
+			<Item label={I18n.t('OS')} content={roomUser.os} />
+			<Item label={I18n.t('Browser')} content={roomUser.browser} />
+			{/* TODO: Check if this param is outdated */}
+			{/* @ts-ignore */}
+			<CustomFields customFields={roomUser.livechatData} />
 			<Title title={I18n.t('Conversation')} theme={theme} />
-			<Item label={I18n.t('Agent')} content={room.servedBy?.username} theme={theme} />
-			<Item label={I18n.t('Facebook')} content={room.facebook?.page.name} theme={theme} />
-			<Item label={I18n.t('SMS')} content={room.sms && 'SMS Enabled'} theme={theme} />
-			<Item label={I18n.t('Topic')} content={room.topic} theme={theme} />
-			<Item label={I18n.t('Tags')} content={room.tags?.join(', ')} theme={theme} />
-			<Item label={I18n.t('Department')} content={department.name} theme={theme} />
-			<CustomFields customFields={room.livechatData} theme={theme} />
+			<Item label={I18n.t('Agent')} content={room.servedBy?.username} />
+			{/* TODO: Check if this param is outdated */}
+			{/* @ts-ignore */}
+			<Item label={I18n.t('Facebook')} content={room.facebook?.page.name} />
```

Comment:
I couldn't find this (or sms) on main repo definitions.
Is this a custom column or is it for Omnichannel only?
If you don't know, can you get this info from Omnichannel team?